nx-X11/config/cf/lnxLib.rules

```diff
@@ -9,7 +9,7 @@
 #endif
 
 #ifndef UseRpath
-#define UseRpath NO
+#define UseRpath YES
```

Comment:
I have wondering for a while, what you mean by the above.

Indeed, it is so, that nxagent is linked with -Wl,-rpath-link,$(BUILDLIBDIR) (because UseInstalled is not defined at build time). This is for the linker, so that it can find all necessary libs for linking nxagent at build time.

And(!), nxagent is liniked with  -Wl,-rpath,$(USRLIBDIRPATH), this is also what we want. This path gets hard-coded into the nxagent executable, so that at runtime the library can be found.

The latter RPATH setting is not conditional to UseInstalled or any other imake variable.